app/templates/error.hbs

```diff
@@ -7,7 +7,7 @@
       <p>
         Indien dit probleem zich blijft voordoen, contacteer
         <AuLinkExternal
-          href="mailto:LoketLokaalBestuur@vlaanderen.be?subject=Technisch probleem in Erediensten Toezichtsdatabank"
+          href="mailto:LoketLokaalBestuur@vlaanderen.be?subject=Technisch probleem in Databank Erediensten"
```

Comment:
Nitpick: You could use the app-name helper here.
```suggestion
          href="mailto:LoketLokaalBestuur@vlaanderen.be?subject=Technisch probleem in {{(app-name)}}"
```

Reply:
Nice catch!